docs/INSTALL.md

```diff
@@ -70,13 +72,13 @@ git submodule update --init --recursive
 Configure the repository for the debug mode:
 
 ```bash
-./configure debug
+./configure.sh debug
```

Comment:
No, debug is not supported anymore. Please check the updated configure script, it accepts a path to the repo with private keys, or runs without parameters for the open-source version.